cargo-dist/src/tasks.rs

```diff
@@ -1373,6 +1390,16 @@ impl<'pkg_graph> DistGraphBuilder<'pkg_graph> {
 
         let formula_name = to_class_case(&app_name);
 
+        let dependencies: Vec<String> = self
+            .inner
+            .system_dependencies
+            .homebrew
+            .clone()
+            .into_iter()
+            .filter(|(_, package)| package.0.stage_wanted(&DependencyKind::Run))
+            .map(|(name, _)| name)
+            .collect();
+
```

Comment:
We could be making multiple homebrew packages (multiple packages in the workspace, each getting its own rb), and ideally each one precisely only depends on what it needs, yeah?